test/screens/v2/widget_instance/reconstructed_alert_test.exs

```diff
@@ -221,12 +225,25 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlertTest do
       assert :reconstructed_large_alert == WidgetInstance.widget_type(widget)
     end
 
-    test "returns flex zone alert for a boundary suspension", %{widget: widget} do
-      widget = put_informed_entities(widget, [ie(stop: "place-dwnxg"), ie(stop: "place-pktrm")])
-      widget = put_effect(widget, :suspension)
-      assert [3] == WidgetInstance.priority(widget)
-      assert [:large] == WidgetInstance.slot_names(widget)
-      assert :reconstructed_large_alert == WidgetInstance.widget_type(widget)
```

Comment:
Did we lose a test for "returns flex zone alert for a boundary suspension at a NON-terminus"? See other removed tests

Reply:
Woops! Sorry about that. Saw an opportunity for some additional test cases as well.